.devcontainer/devcontainer.json

```diff
@@ -1,21 +1,34 @@
 {
+  // See the devcontainer spec at https://containers.dev/implementors/spec/
+  "name": "React Codespaces Template",
+
+  // Container image to build environment from
+  // universal contains many tools making it useful in many contexts https://github.com/devcontainers/images/tree/main/src/universal
   "image": "mcr.microsoft.com/devcontainers/universal:2",
+
+  // Specify the minimum machine spec for running the dev container
   "hostRequirements": {
     "cpus": 4
   },
-  "waitFor": "onCreateCommand",
```

Comment:
My understanding is that using this instead of the default `updateContentCommand` means we can connect sooner, while we're installing dependencies, as opposed to waiting for dependencies to be installed.

That's _mostly_ a moot point, given that we're using prebuilds but still seems valuable?

Reply:
I don't see how it is valuable if we're using Codespaces prebuilds. Help me understand

Reply:
Two thoughts:
1. If prebuilds are broken or stale, it helps
2. If folks are copying & pasting the devcontainer from these templates to use in their own projects, this is a reasonable default

Reply:
Makes sense. What would you provide as a comment inline to explain the wait for an empty command?

Reply:
How about: "Allow connecting earlier, while we're still installing dependencies"